pkg/operator/controller/clientca-configmap/controller_test.go

```diff
@@ -44,7 +44,7 @@ import (
 // suitable fake IngressController and configmap objects for the various test
 // cases to verify that the controller correctly performs the above tasks.
 func Test_Reconcile(t *testing.T) {
-	exampleTime, err := time.Parse(time.DateTime, "2006-01-02 15:04:05")
+	exampleTime, err := time.Parse(time.RFC3339, "2006-01-02T15:04:05Z")
```

Comment:
The other commit in this PR added this test, so it would make more sense (and be more `git bisect`-friendly) to edit that commit instead of making this change in this commit.  Also, we have been using `Modified-by:` in commit messages to indicate when someone has modified what the commit author wrote.

Reply:
Fixed in https://github.com/openshift/cluster-ingress-operator/commit/d2126f8ce2485392890dc24680760bdcbd47d06b